src/shared/backgroundScripts/polyfill.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Polyfill the browser object to be compatible with both Chrome MV3 and Firefox MV2.
+ */
+export function polyfillBrowser() {
```

Comment:
You don't need to export a function. You can import it elsewhere with `import "polyfill.js";` without any named imports.